js/common.js

```diff
@@ -386,7 +386,9 @@ $(document).ready( function() {
 
 	$(document).on('shown.bs.dropdown', '#dropdown_projects_menu', function() {
 		$(this).find(".dropdown-menu li.active a").focus();
-	 });
+		/* Add autofocus support for IE */
+		$('[autofocus]:not(:focus)').eq(0).focus();
```

Comment:
According to Can I use [autofocus attribute is supported since IE 10](https://caniuse.com/#feat=autofocus), and our minimum req is IE11 so I'm not sure this is actually needed.

Reply:
Sadly Internet Explorer does not support autofocus at all. Despite it saying so in 'Can I Use'. So this is indeed needed.

Reply:
There was also an MSDN article saying that it was supported on IE10. Anyway, I don't use IE so I'll trust you on that one.